manifest.go

```diff
@@ -33,6 +33,7 @@ var (
 	errInvalidMetadata     = errors.New("metadata should be a TOML table")
 
 	errInvalidProjectRoot = errors.New("ProjectRoot name validation failed")
+	errInefficientRules   = errors.New("Inefficient rules found in the manifest")
```

Comment:
errors should start with lowercase, unless they are some special term, like `ProjectRoot`.